src/current/_includes/v24.1/sql/disallow-full-table-scans.md

```diff
@@ -0,0 +1,12 @@
+- At the cluster level, set `disallow_full_table_scans` for some or all users/roles. For example:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    ALTER ROLE ALL SET disallow_full_table_scans = true;
+    ~~~
+
+- At the application level, add `disallow_full_table_scans` to the connection string using the [`options` parameter]({% link {{page.version.version}}/connection-parameters.md %}#additional-connection-parameters).
```

Comment:
Is it worth giving an example?

Reply:
Perhaps, but since this PR is largely moving existing docs around, I think I'd prefer to move ahead for now.